.github/workflows/main.yml

```
@@ -8,6 +8,33 @@ on:
 env:
   DEBIAN_FRONTEND: noninteractive
 jobs:
+  luacheck:
+    name: Check code style
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install additional packages
+        run: |
+          set -ex
+          sudo apt -qy update
+          sudo apt -qy install --no-install-recommends lua-check
+      - name: Run luacheck
+        run: find */src/ -type f -name '*.lua' -follow -exec luacheck {} +
```

Include `build.lua` and the testfiles in the check.
```suggestion
        run: find -type f -name '*.lua' -follow -exec luacheck {} +
```